CHANGELOG.md

```diff
@@ -11,11 +11,18 @@ All user visible changes to this project will be documented in this file. This p
 
 [Diff](/../../compare/v0.1.0-alpha.13.5...v0.1.0-alpha.14) | [Milestone](/../../milestone/24)
 
+### BC Breaks
+
+- L10n:
+    - Replaced `email_verification_code` and `email_verification_code_subject` Fluent phrases with `email_confirmation_code` and `email_confirmation_code_subject` (#1022).
+
 ### Added
 
 - UI:
     - Home page:
         - Link tab. ([#1012])
+- L10n:
+    - `fcm_webpush_link` Fluent phrase (#1022).
```

Comment:
@50U10FCA7, I wonder whether the listed changes should be considered as visible to the end user and thus listed there at all. Users won't notice the phrases being added or renamed, as this won't be reflected in the final product visually. Furthermore, phrases being added, renamed or replaced might spam the change log completely, as these kind of changes happen all the time. What're your thoughts? Don't you mind omitting this here?